### PR TITLE
a bunch of updates

### PR DESCRIPTION
**Pull request by adamdruppe**
The website has some old links and could use just a little more information so people stop claiming gdc is deadl

### CODE REVIEW

File: views/downloads.md [hunk 1]
```diff
@@ -5,20 +5,26 @@ For access to the current development sources of GDC, visit
...
+ * [Work-in-progress unofficial Windows release](http://www.winlibs.com/)
 
-[gitrepo]: https://github.com/D-Programming-GDC/GDC
+[gitrepo]: https://github.com/D-Programming-GDC/gcc
 [installwiki]: http://wiki.dlang.org/GDC/Installation
-[archpkg]: https://www.archlinux.org/packages/?sort=&arch=i686&arch=x86_64&q=gdc
-[debianpkg]: https://packages.debian.org/search?keywords=gdc&searchon=names&suite=all&section=all
-[ubuntupkg]: http://packages.ubuntu.com/search?keywords=gdc&searchon=names&exact=1&suite=all&section=all
+[archpkg]: https://archlinux.org/packages/core/x86_64/gcc-d/
+[debianpkg]: https://packages.debian.org/stable/devel/gdc-8
```
Comment (ibuclaw) [inline views/downloads.md:26]:
```suggestion
[debianpkg]: https://packages.debian.org/stable/devel/gdc
```

File: views/index.md [hunk 2]
```diff
@@ -41,7 +43,7 @@ are part of preparing a new improved version.  For general help with GDC,
 the [D.gnu][mailinglist] mailing list is the place to go with questions
 or problems.
 
-[bugzilla]: http://bugzilla.gdcproject.org
-[openbugs]: http://bugzilla.gdcproject.org/describecomponents.cgi
+[bugzilla]: https://gcc.gnu.org/bugzilla/
+[openbugs]: https://gcc.gnu.org/bugzilla/describecomponents.cgi
```
Comment (ibuclaw) [inline views/index.md:47]:
```suggestion
[openbugs]: https://gcc.gnu.org/bugzilla/buglist.cgi?product=gcc&component=d&resolution=---
```

File: templates/menu.inc [hunk 3]
```diff
@@ -3,8 +3,8 @@
   <a class="nav-item nav-link ml-3" href="/contributing">contributing</a>
   <a class="nav-item nav-link ml-3" href="/downloads">downloads</a>
   <a class="nav-item nav-link ml-3" href="/documentation">documentation</a>
-  <a class="nav-item nav-link ml-3" href="http://bugzilla.gdcproject.org">bugs</a>
-  <a class="nav-item nav-link ml-3" href="https://github.com/D-Programming-GDC/GDC">current&nbsp;git</a>
+  <a class="nav-item nav-link ml-3" href="https://gcc.gnu.org/bugzilla/buglist.cgi?quicksearch=gdc">bugs</a>
```
Comment (ibuclaw) [inline templates/menu.inc:6]:
```suggestion
  <a class="nav-item nav-link ml-3" href="https://gcc.gnu.org/bugzilla/">bugs</a>
```
If you really insist on only listing gdc-specific bugs, then use the `[openbugs]` URL I suggested below.

**Comment by ibuclaw** [comment]
> Continuous integration has not been set up 

I should create an issue on this so as to not forget. ;-)

File: views/index.md [hunk 4]
```diff
@@ -14,6 +14,8 @@ now under the lead of Iain Buclaw, who has been steering the project since
 2009 with the assistance of its contributors.  Without them, the project
 would not have been nearly as successful as it has been.
 
+GDC was merged into the upstream GCC tree in GCC 9.0 and continues to be developed and distributed as part of GCC.
```
Comment (ibuclaw) [inline views/index.md:17]: You can use `gqq` in vim to break up the line, but I'm not really too bothered about line lengths.

File: templates/menu.inc (hunk 3 above)
Comment (adamdruppe) [reply templates/menu.inc]: k changed

**Review by ibuclaw — APPROVED** [review]

**Comment by adamdruppe** [comment]
btw im trying to convince my windows box to build gdc right now so i can actually help out on that.

but i have no gnu make :( still ill dl that and hopefully have it running w/ your one patch p soon and can see for myself if it actually works

**Comment by ibuclaw** [comment]
> btw im trying to convince my windows box to build gdc right now so i can actually help out on that.
> 
> but i have no gnu make :( still ill dl that and hopefully have it running w/ your one patch p soon and can see for myself if it actually works

Cygwin may be a good place to start, MinGW is quite awkward by comparison and really outdated, though I'm going off memory from circa 2009.  A cross compiler from WSL might be a new possibility as well.

**Comment by ibuclaw** [comment]
By the way, thanks so much for doing this @adamdruppe.

**Comment by adamdruppe** [comment]
On Fri, Mar 19, 2021 at 09:56:05AM -0700, Iain Buclaw wrote:
> By the way, thanks so much for doing this @adamdruppe.

gdc rox

**Pull request merged**